src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,49 @@
 import React from 'react';
+import { PreparedPost } from '../../types/PreparedPost';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+interface Props {
+  post: PreparedPost;
+}
+
+const NoComments = () => (
```

Comment:
No need to move this to a separate variable bc it variable would be render each time
Move this to the return